client/app/telecom/telephony/alias/configuration/feature/conference/telecom-telephony-alias-configuration-conference.controller.js

```diff
@@ -0,0 +1,180 @@
+angular.module('managerApp').controller('TelecomTelephonyAliasConfigurationConferenceCtrl', class TelecomTelephonyAliasConfigurationContactCenterSolutionCtrl {
+  constructor(
+    $q, $state, $stateParams, $timeout, $translate,
+    TucToast, tucVoipServiceAlias,
+    TUC_TELEPHONY_ALIAS, TELEPHONY_SERVICE,
+  ) {
+    this.$q = $q;
+    this.$state = $state;
+    this.$stateParams = $stateParams;
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.TucToast = TucToast;
+    this.tucVoipServiceAlias = tucVoipServiceAlias;
+    this.TUC_TELEPHONY_ALIAS = TUC_TELEPHONY_ALIAS;
+    this.TELEPHONY_SERVICE = TELEPHONY_SERVICE;
+  }
+
+  $onInit() {
+    this.conferenceCopy = null;
+    this.generatingUrls = false;
+    this.hasWebAccessUrls = false;
+    this.newFile = null;
+    this.availableLanguages = this.TUC_TELEPHONY_ALIAS.conference.languages
+      .map(({ label, value }) => ({
+        label: this.$translate.instant(`language_${label}`),
+        value,
+      }));
+    this.availableReportStatus = this.TUC_TELEPHONY_ALIAS.conference.reportStatus.map(status => ({
+      label: this.$translate.instant(`telephony_alias_config_conference_report_status_${status}`),
+      value: status,
+    }));
+
+    this.serviceInfos = {
+      billingAccount: this.$stateParams.billingAccount,
+      serviceName: this.$stateParams.serviceName,
+    };
+
+    this.loading = true;
+    return this.tucVoipServiceAlias.fetchConferenceNumber(this.serviceInfos)
+      .then((conference) => {
+        this.conference = _.omit(conference, ['$promise', '$resolved']);
```

Comment:
Why not use `angular.copy` instead?

Reply:
```suggestion
        { $promis, $resolved, ...this.conference } = conference;
```

Reply:
IMOH I find `_.omit` easier to understand. I know we're trying to get rid of underscore in the long run but, it just conveys meaning better than declaring some variables and not using them, I guess.